packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -4,24 +4,49 @@ import { lock } from '@wordpress/icons';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
 
-interface DomainsTableSSLCellProps {
+interface DomainsTableSslCellProps {
 	domainManagementLink: string;
 	sslStatus: DomainData[ 'ssl_status' ];
+	hasWpcomManagedSslCert?: DomainData[ 'has_wpcom_managed_ssl_cert' ];
 }
 
-export default function DomainsTableSSLCell( {
+export default function DomainsTableSslCell( {
 	domainManagementLink,
 	sslStatus,
-}: DomainsTableSSLCellProps ) {
+	hasWpcomManagedSslCert,
+}: DomainsTableSslCellProps ) {
 	const translate = useTranslate();
+	// WordPress.com managed subdomains (e.g. *.wordpress.com, *.wpcomstaging.com, etc.)
+	// are covered by a wildcard SSL cert, so we display them as 'Active'.
+	const isActiveSsl = sslStatus === 'active' || hasWpcomManagedSslCert;
+	const isPendingSsl = sslStatus === 'pending' || sslStatus === 'newly_registered';
+	const domainHasSsl = sslStatus || hasWpcomManagedSslCert;
```

Comment:
```suggestion
	const domainHasSsl = sslStatus !== null || hasWpcomManagedSslCert;
```
Nit, but we might as well be explicit about which value we're looking for here.